raiden_contracts/utils/remove_limits.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+
+import click
+from pathlib import Path
+import subprocess
+import re
+
+from raiden_contracts.constants import CONTRACTS_VERSION, CONTRACTS_VERSION_LIMITED
+
+"""
+A utility to copy a directory containing Solidity files and
```

Comment:
nitpick: /s/A/An

Reply:
I don't see it as an "an". "A utility" sounds valid for me.

Reply:
It's not a matter of how you see it or how it sounds to you. It's the language's grammar. Before a vowel `a` become `an`. https://www.englishforums.com/English/AUtilityVsAnUtility/hrmlw/post.htm

Reply:
True... but doesn't "An utility" sound wrong to you?

Reply:
A vowel is a sound.  It's not a charecter.  `utility` sounds like `[yoo-til-i-tee]` so it takes an `a`.

Reply:
> A vowel is a sound

Vowell are letters. The English language has 5 of them. http://www.abcfastphonics.com/vowel-letters.html

But indeed the rule is for vowel letters and there are exception for some words starting with `u` depending on how they sound. So it is indeed a utility, just like a university. as you explained. "yoo-".

Reply:
Now Augusto sent me https://www.englishclub.com/pronunciation/a-an.htm

Reply:
Ha nice link with many examples :)